docs/getting-started/quick-start-guides/deploy-rancher-manager/helm-cli.md

```diff
@@ -15,14 +15,14 @@ The full installation requirements are [here](../../../pages-for-subheaders/inst
 
 ## Install K3s on Linux
 
+Rancher needs to be installed on a supported Kubernetes version. To specify the K3s version, use the INSTALL_K3S_VERSION environment variable when running the K3s installation script. Refer to the [support maintenance terms](https://rancher.com/support-maintenance-terms/).
```

Comment:
```suggestion
Rancher needs to be installed on a supported Kubernetes version. To specify the K3s version, use the INSTALL_K3S_VERSION (e.g., `INSTALL_K3S_VERSION="v1.24.10+k3s1"`) environment variable when running the K3s installation script. Refer to the [support maintenance terms](https://rancher.com/support-maintenance-terms/).
```

Using 1.24.10 instead of proposed 1.24.11 since 1.24.10 is what https://www.suse.com/suse-k3s/support-matrix/all-supported-versions/k3s-v1-24/ currently lists.